packages/main/src/plugin/extension/extension-loader.ts

```diff
@@ -738,7 +732,7 @@ export class ExtensionLoader {
 
         telemetryOptions['loadingRuntimeDuration'] = afterLoadingRuntime - beforeLoadingRuntime;
 
-        await this.activateExtension(extension, runtime);
+        await this.activateExtension(extension, runtime as unknown as Record<string, unknown>);
```

Comment:
question: how is it possible to cast runtime to unknown and then Record<string, unknown> without any check ?

Reply:
Fixed by a216667854010b70744bb842584a1770ac90fa1d